### ISSUE TITLE
Debugger: Assertion failed when running on non-existent executable

### ISSUE THREAD

**Issue by sppmacd** [SerenityOS/serenity#5042]
```
USERSPACE(45) ASSERTION FAILED: false
./Userland/Libraries/LibDebug/DebugSession.cpp:88
[Debugger(45:45)]: 0x00000000  (?)

[Debugger(45:45)]: 0xc018380c  Kernel::Process::crash(int, unsigned int, bool) +0x2a8
[Debugger(45:45)]: 0xc01895d7  Kernel::Process::sys$abort() +0x1f
[Debugger(45:45)]: 0xc0188aab  syscall_handler +0x6be
[Debugger(45:45)]: 0xc01883e5  syscall_asm_entry +0x31
[Debugger(45:45)]: Process regions:
[Debugger(45:45)]: BEGIN       END         SIZE        ACCESS  NAME
[Debugger(45:45)]: 01601000 -- 01a00fff    00400000    RW  TA    Stack (Main thread)
[Debugger(45:45)]: 01a0d000 -- 01a0dfff    00001000    RW   A    (null)
[Debugger(45:45)]: 01a10000 -- 01a1ffff    00010000         A    malloc: ChunkedBlock(256)
[Debugger(45:45)]: 01a21000 -- 01a22fff    00002000    RW   A    Thread-specific
[Debugger(45:45)]: 01a24000 -- 01a25fff    00002000    R XS      libm.so: .text
[Debugger(45:45)]: 01a26000 -- 01a26fff    00001000    RW   A    libm.so: .data
[Debugger(45:45)]: 01a28000 -- 01a28fff    00001000    R XS      libcrypt.so: .text
[Debugger(45:45)]: 01a29000 -- 01a29fff    00001000    RW   A    libcrypt.so: .data
[Debugger(45:45)]: 01a30000 -- 01a3ffff    00010000    RW   A    malloc: ChunkedBlock(32)
[Debugger(45:45)]: 01a41000 -- 01a46fff    00006000    R XS      /bin/Debugger: .text
[Debugger(45:45)]: 01a47000 -- 01a47fff    00001000    RW   A    /bin/Debugger: .data
[Debugger(45:45)]: 01a50000 -- 01a5ffff    00010000    RW   A    malloc: ChunkedBlock(128)
[Debugger(45:45)]: 01a70000 -- 01a7ffff    00010000    RW   A    malloc: ChunkedBlock(64)
[Debugger(45:45)]: 01a81000 -- 01a99fff    00019000    R         ELF_DYN: /bin/Debugger
[Debugger(45:45)]: 01aa0000 -- 01aaffff    00010000    RW   A    malloc: ChunkedBlock(256)
[Debugger(45:45)]: 01ac0000 -- 01acffff    00010000    RW   A    malloc: ChunkedBlock(500)
[Debugger(45:45)]: 01ae0000 -- 01aeffff    00010000    RW   A    malloc: ChunkedBlock(1016)
[Debugger(45:45)]: 01b00000 -- 01b0ffff    00010000    RW   A    malloc: ChunkedBlock(500)
[Debugger(45:45)]: 01b20000 -- 01b2ffff    00010000    RW   A    malloc: ChunkedBlock(32)
[Debugger(45:45)]: 01b40000 -- 01b4ffff    00010000    RW   A    malloc: ChunkedBlock(64)
[Debugger(45:45)]: 01b60000 -- 01b6ffff    00010000    RW   A    malloc: ChunkedBlock(128)
[Debugger(45:45)]: 01b80000 -- 01b8ffff    00010000    RW   A    malloc: ChunkedBlock(8)
[Debugger(45:45)]: 01ba0000 -- 01baffff    00010000    RW   A    malloc: ChunkedBlock(16)
[Debugger(45:45)]: 01bc0000 -- 01bcffff    00010000    RW   A    malloc: ChunkedBlock(16376)
[Debugger(45:45)]: 01f99000 -- 01fb0fff    00018000    R XS      libgcc_s.so: .text
[Debugger(45:45)]: 01fb1000 -- 01fb1fff    00001000    RW   A    libgcc_s.so: .data
[Debugger(45:45)]: 01fc0000 -- 01fcffff    00010000         A    malloc: ChunkedBlock(16)
[Debugger(45:45)]: 01fe0000 -- 01feffff    00010000         A    malloc: ChunkedBlock(2032)
[Debugger(45:45)]: 01ff1000 -- 02053fff    00063000    R X       libc.so: .text
[Debugger(45:45)]: 02054000 -- 0205afff    00007000    RW   A    libc.so: .data
[Debugger(45:45)]: 0205c000 -- 02072fff    00017000    R XS      libcrypto.so: .text
[Debugger(45:45)]: 02073000 -- 02073fff    00001000    RW   A    libcrypto.so: .data
[Debugger(45:45)]: 02075000 -- 020a9fff    00035000    R XS      libcore.so: .text
[Debugger(45:45)]: 020aa000 -- 020abfff    00002000    RW   A    libcore.so: .data
[Debugger(45:45)]: 020b0000 -- 020bffff    00010000         A    malloc: ChunkedBlock(4088)
[Debugger(45:45)]: 020c1000 -- 020e5fff    00025000    R XS      libregex.so: .text
[Debugger(45:45)]: 020e6000 -- 020e6fff    00001000    RW   A    libregex.so: .data
[Debugger(45:45)]: 020e8000 -- 020f9fff    00012000    R XS      libdebug.so: .text
[Debugger(45:45)]: 020fa000 -- 020fafff    00001000    RW   A    libdebug.so: .data
[Debugger(45:45)]: 020fc000 -- 0210afff    0000f000    R XS      libx86.so: .text
[Debugger(45:45)]: 0210b000 -- 02115fff    0000b000    RW   A    libx86.so: .data
[Debugger(45:45)]: 02117000 -- 02134fff    0001e000    R XS      libline.so: .text
[Debugger(45:45)]: 02135000 -- 02135fff    00001000    RW   A    libline.so: .data
[Debugger(45:45)]: 126cc000 -- 1270dfff    00042000    R XS      /usr/lib/Loader.so
[Debugger(45:45)]: 1270e000 -- 12713fff    00006000    RW   A    /usr/lib/Loader.so (data-rw)
[Debugger(45:45)]: Kernel regions:
[Debugger(45:45)]: BEGIN       END         SIZE        ACCESS  NAME
[Debugger(45:45)]: c0904000 -- c0904fff    00001000    RW   A    HPET MMIO
[Debugger(45:45)]: c0906000 -- c0906fff    00001000    R X  A    Signal trampolines
[Debugger(45:45)]: c0908000 -- c0917fff    00010000    RW  TA    Kernel Stack (Thread 0)
[Debugger(45:45)]: c092a000 -- c0939fff    00010000    RW  TA    Kernel Stack (Thread 2)
[Debugger(45:45)]: c093b000 -- c094afff    00010000    RW  TA    Kernel Stack (Thread 3)
[Debugger(45:45)]: c094c000 -- c094cfff    00001000     W        UHCI Framelist
[Debugger(45:45)]: c094e000 -- c094ffff    00002000     W        UHCI Queue Head Pool
[Debugger(45:45)]: c0951000 -- c0952fff    00002000     W        UHCI Transfer Descriptor Pool
[Debugger(45:45)]: c0954000 -- c0963fff    00010000    RW  TA    Kernel Stack (Thread 4)
[Debugger(45:45)]: c0965000 -- c0965fff    00001000    RW        E1000 RX
[Debugger(45:45)]: c0967000 -- c0967fff    00001000    RW        E1000 TX
[Debugger(45:45)]: c0969000 -- c0988fff    00020000    RW   A    E1000 MMIO
[Debugger(45:45)]: c098a000 -- c098bfff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c098d000 -- c098efff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c0990000 -- c0991fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c0993000 -- c0994fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c0996000 -- c0997fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c0999000 -- c099afff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c099c000 -- c099dfff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c099f000 -- c09a0fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09a2000 -- c09a3fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09a5000 -- c09a6fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09a8000 -- c09a9fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09ab000 -- c09acfff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09ae000 -- c09affff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09b1000 -- c09b2fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09b4000 -- c09b5fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09b7000 -- c09b8fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09ba000 -- c09bbfff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09bd000 -- c09befff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09c0000 -- c09c1fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09c3000 -- c09c4fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09c6000 -- c09c7fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09c9000 -- c09cafff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09cc000 -- c09cdfff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09cf000 -- c09d0fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09d2000 -- c09d3fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09d5000 -- c09d6fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09d8000 -- c09d9fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09db000 -- c09dcfff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09de000 -- c09dffff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09e1000 -- c09e2fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09e4000 -- c09e5fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09e7000 -- c09e8fff    00002000    RW        E1000 RX buffer
[Debugger(45:45)]: c09ea000 -- c09ebfff    00002000    RW        E1000 TX buffer
[Debugger(45:45)]: c09ed000 -- c09eefff    00002000    RW        E1000 TX buffer
[Debugger(45:45)]: c09f0000 -- c09f1fff    00002000    RW        E1000 TX buffer
[Debugger(45:45)]: c09f3000 -- c09f4fff    00002000    RW        E1000 TX buffer
[Debugger(45:45)]: c09f6000 -- c09f7fff    00002000    RW        E1000 TX buffer
[Debugger(45:45)]: c09f9000 -- c09fafff    00002000    RW        E1000 TX buffer
[Debugger(45:45)]: c09fc000 -- c09fdfff    00002000    RW        E1000 TX buffer
[Debugger(45:45)]: c09ff000 -- c0a00fff    00002000    RW        E1000 TX buffer
[Debugger(45:45)]: c0a02000 -- c0a02fff    00001000    RW   A    Ext2FS: Block group descriptors
[Debugger(45:45)]: c0a04000 -- c3113fff    02710000    RW   A    KBuffer
[Debugger(45:45)]: c3115000 -- c314ffff    0003b000    RW   A    KBuffer
[Debugger(45:45)]: c3151000 -- c3151fff    00001000    RW   A    Ext2FS: Cached bitmap block
[Debugger(45:45)]: c3153000 -- c3162fff    00010000    RW  TA    Kernel Stack (Thread 5)
[Debugger(45:45)]: c3164000 -- c3164fff    00001000    RW   A    Ext2FS: Cached bitmap block
[Debugger(45:45)]: c3166000 -- c3166fff    00001000    RW   A    Ext2FS: Cached bitmap block
[Debugger(45:45)]: c3168000 -- c3177fff    00010000    RW  TA    Kernel Stack (Thread 6)
[Debugger(45:45)]: c3179000 -- c3188fff    00010000    RW   A    Kernel Packet Buffer
[Debugger(45:45)]: c318a000 -- c31a9fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c31ab000 -- c31cafff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c31cc000 -- c31ebfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c31ed000 -- c320cfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c320e000 -- c322dfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c322f000 -- c324efff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3250000 -- c326ffff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3271000 -- c3290fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3292000 -- c32b1fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c32b3000 -- c32d2fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c32d4000 -- c32f3fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c32f5000 -- c3314fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3316000 -- c3335fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3337000 -- c3356fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3358000 -- c3377fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3379000 -- c3398fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c339a000 -- c33b9fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c33bb000 -- c33dafff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c33dc000 -- c33fbfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c33fd000 -- c341cfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c341e000 -- c343dfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c343f000 -- c345efff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3460000 -- c347ffff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3481000 -- c34a0fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c34a2000 -- c34c1fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c34c3000 -- c34d2fff    00010000    RW  TA    Kernel Stack (Thread 8)
[Debugger(45:45)]: c34d4000 -- c34e3fff    00010000    RW  TA    Kernel Stack (Thread 9)
[Debugger(45:45)]: c34e5000 -- c34f4fff    00010000    RW  TA    Kernel Stack (Thread 10)
[Debugger(45:45)]: c34f6000 -- c3505fff    00010000    RW  TA    Kernel Stack (Thread 11)
[Debugger(45:45)]: c3507000 -- c3516fff    00010000    RW  TA    Kernel Stack (Thread 12)
[Debugger(45:45)]: c3518000 -- c3527fff    00010000    RW  TA    Kernel Stack (Thread 13)
[Debugger(45:45)]: c3529000 -- c3538fff    00010000    RW  TA    Kernel Stack (Thread 14)
[Debugger(45:45)]: c353a000 -- c3549fff    00010000    RW  TA    Kernel Stack (Thread 15)
[Debugger(45:45)]: c354b000 -- c355afff    00010000    RW  TA    Kernel Stack (Thread 16)
[Debugger(45:45)]: c355c000 -- c356bfff    00010000    RW  TA    Kernel Stack (Thread 17)
[Debugger(45:45)]: c356d000 -- c357cfff    00010000    RW  TA    Kernel Stack (Thread 18)
[Debugger(45:45)]: c357e000 -- c358dfff    00010000    RW  TA    Kernel Stack (Thread 19)
[Debugger(45:45)]: c358f000 -- c359efff    00010000    RW  TA    Kernel Stack (Thread 20)
[Debugger(45:45)]: c35a0000 -- c35affff    00010000    RW  TA    Kernel Stack (Thread 21)
[Debugger(45:45)]: c35b1000 -- c35c0fff    00010000    RW  TA    Kernel Stack (Thread 22)
[Debugger(45:45)]: c35c2000 -- c35d1fff    00010000    RW  TA    Kernel Stack (Thread 23)
[Debugger(45:45)]: c35d3000 -- c35f2fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c35f4000 -- c3613fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3615000 -- c3634fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3636000 -- c3645fff    00010000    RW  TA    Kernel Stack (Thread 24)
[Debugger(45:45)]: c3647000 -- c3666fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3668000 -- c3687fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3689000 -- c36a8fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c36aa000 -- c36c9fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c36cb000 -- c36eafff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c36ec000 -- c370bfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c370d000 -- c372cfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c372e000 -- c374dfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c374f000 -- c376efff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3770000 -- c378ffff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3791000 -- c37b0fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c37b2000 -- c37d1fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c37d3000 -- c37f2fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c37f4000 -- c3813fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3815000 -- c3834fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3836000 -- c3855fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3857000 -- c3876fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3878000 -- c3897fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3899000 -- c38b8fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c38ba000 -- c38d9fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c38db000 -- c38fafff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c38fc000 -- c391bfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c391d000 -- c393cfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c393e000 -- c395dfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c395f000 -- c397efff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3980000 -- c399ffff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c39a1000 -- c39c0fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c39c2000 -- c39e1fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c39e3000 -- c3a02fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3a04000 -- c3a23fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3a25000 -- c3a44fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3a46000 -- c3a65fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3a67000 -- c3e66fff    00400000    RW   A    KBuffer
[Debugger(45:45)]: c3e68000 -- c3e87fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3e89000 -- c3ea8fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3eaa000 -- c3ec9fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3ecb000 -- c3eeafff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3eec000 -- c3f0bfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3f0d000 -- c3f2cfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3f2e000 -- c3f4dfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3f4f000 -- c3f6efff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3f70000 -- c3f8ffff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3f91000 -- c3fb0fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3fb2000 -- c3fd1fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3fd3000 -- c3ff2fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c3ff4000 -- c4013fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4015000 -- c4034fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4036000 -- c4055fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4057000 -- c4076fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4078000 -- c4097fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4099000 -- c40b8fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c40ba000 -- c40d9fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c40db000 -- c40fafff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c40fc000 -- c411bfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c411d000 -- c413cfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c413e000 -- c415dfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c415f000 -- c417efff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4180000 -- c419ffff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c41a1000 -- c41c0fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c41c2000 -- c41e1fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c41e3000 -- c4202fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4204000 -- c4223fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4225000 -- c4244fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4246000 -- c4265fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4267000 -- c4286fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4288000 -- c42a7fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c42a9000 -- c42c8fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c42ca000 -- c42e9fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c42eb000 -- c430afff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c430c000 -- c432bfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c432d000 -- c434cfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c434e000 -- c436dfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c436f000 -- c438efff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4390000 -- c43affff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c43b1000 -- c43c0fff    00010000    RW  TA    Kernel Stack (Thread 25)
[Debugger(45:45)]: c43d3000 -- c43f2fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c43f4000 -- c4413fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4415000 -- c4434fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4436000 -- c4455fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4457000 -- c4476fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4478000 -- c4497fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4499000 -- c44b8fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c091b000 -- c091bfff    00001000    RW   A    KBuffer
[Debugger(45:45)]: c44ba000 -- c44d9fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c44db000 -- c44fafff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c44fc000 -- c451bfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c451d000 -- c453cfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c453e000 -- c455dfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c455f000 -- c457efff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4580000 -- c459ffff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c45a1000 -- c45c0fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c45c2000 -- c45e1fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c45e3000 -- c4602fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4604000 -- c4623fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4625000 -- c4644fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4646000 -- c4665fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4667000 -- c4686fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c4688000 -- c46a7fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c46a9000 -- c46c8fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c091d000 -- c091dfff    00001000    RW   A    Ext2FS: Cached bitmap block
[Debugger(45:45)]: c0801000 -- c0900fff    00100000    RW   A    kmalloc subheap
[Debugger(45:45)]: c46ca000 -- c47c9fff    00100000    RW   A    kmalloc subheap
[Debugger(45:45)]: c47cb000 -- c4bcafff    00400000    RW   A    KBuffer
[Debugger(45:45)]: c4bcc000 -- c4fcbfff    00400000    RW   A    KBuffer
[Debugger(45:45)]: c4fff000 -- c501efff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c5020000 -- c503ffff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c091f000 -- c091ffff    00001000    RW   A    Ext2FS: Cached bitmap block
[Debugger(45:45)]: c50c5000 -- c50d4fff    00010000    RW  TA    Kernel Stack (Thread 28)
[Debugger(45:45)]: c0921000 -- c0921fff    00001000    RW   A    Ext2FS: Cached bitmap block
[Debugger(45:45)]: c0923000 -- c0923fff    00001000    RW   A    Ext2FS: Cached bitmap block
[Debugger(45:45)]: c5052000 -- c5071fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c5073000 -- c5092fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c51ad000 -- c51ccfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c51ce000 -- c51edfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c5263000 -- c5282fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c5284000 -- c52a3fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c9285000 -- cd283fff    03fff000    RW   A    KBuffer
[Debugger(45:45)]: c52a5000 -- c53a4fff    00100000    RW   A    kmalloc subheap
[Debugger(45:45)]: c50d6000 -- c50f5fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c50f7000 -- c5116fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c5118000 -- c5137fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c5139000 -- c5158fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c53e8000 -- c5407fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c5409000 -- c5428fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c515a000 -- c5179fff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c517b000 -- c519afff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c548d000 -- c54acfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c54ae000 -- c54cdfff    00020000    RW   A    DoubleBuffer
[Debugger(45:45)]: c5094000 -- c50a3fff    00010000    RW  TA    Kernel Stack (Thread 44)
[Debugger(45:45)]: c4fcd000 -- c4fdcfff    00010000    RW  TA    Kernel Stack (Thread 45)
[Debugger(45:45)]: Debugger(45:45) Should already die
[FinalizerTask(3:3)]: Generating coredump for pid: 45
[FinalizerTask(3:3)]: RTC: Year: 2021, month: 1, day: 22, hour: 15, minute: 27, second: 47
CrashDaemon(20): New coredump file: /tmp/coredump/Debugger_45_1611329267
Debugger(44): perror(): PT_ATTACH: No such process
[Debugger(44:44)]: Debugger(44:44) Should already die
Shell(25): Job entry "Debugger hs" deleted in 287 ms
CrashDaemon(20): --- Backtrace for thread #0 (TID 45) ---
CrashDaemon(20): 0x02008403: [libc.so] __assertion_failed +0x98 (Syscall.h:449)
CrashDaemon(20): 0x020f3ab3: [libdebug.so] Debug::DebugSession::exec_and_attach(AK::String const&, AK::String) +0xd1 (StringImpl.h:63)
CrashDaemon(20): 0x01a42e83: [/bin/Debugger]  +0x203 (main.cpp:209)
CrashDaemon(20): 0x01a447fc: [/bin/Debugger] _start +0x42 (crt0_shared.cpp:60)
```

**Comment by awesomekling** [comment]
`hs` is an alias, not a symlink :)

This can be reproduced with `sdb sdfujsldfjlsdfjlkdf` or any other non-existent executable.

**Comment by linusg** [comment]
Fixed by #5044.
